src/EmuFs.cc

```diff
@@ -86,14 +102,21 @@ void EmuFile::update(dev_t device, ino_t inode, uint64_t size) {
        << "-inode-" << orig_inode << "-" << path_tag;
   string real_name = name.str().substr(0, 255);
 
-  ScopedFd fd =
-      open(real_name.c_str(), O_CREAT | O_EXCL | O_RDWR | O_CLOEXEC, 0600);
+  int res = memfd_create(real_name.c_str(), MFD_CLOEXEC);
```

Comment:
The problem is that /dev/shm is fixed in size to e.g. half the ram and I don't want to tweak this if needed when there is a better interface. I guess it also has the benefit that there is no tmpfs overhead.